src/App.js

```diff
@@ -1,9 +1,53 @@
-function App () {
+import birdData from "./data/birds";
+import bonusItems from "./data/bonusItems";
+import "./App.css";
+import { useState } from "react";
+import Birdcards from "./components/Birdcards";
+import Cart from "./components/Cart";
+import Checkout from "./components/Checkout";
+
+function App() {
+  const [birds, setBirds] = useState([]);
+
+  function setBirdCount(bird) {
+    const allBirds = { ...bird };
+    setBirds([...birds, allBirds]);
+  }
+
   return (
-    <div>
-      <h1>Hello, world!</h1>
+    <div className="App">
+      <header>
+        <h4 className="title">Bird Sanctuary Lab</h4>
+      </header>
+      <main>
+        <div>
+          <Cart birds={birds} bonusItems={bonusItems} setBirds={setBirds} />
+          <Checkout setBirds={setBirds} />
+        </div>
+        <Birdcards dataBirds={birdData} setBirdCount={setBirdCount} />
```

Comment:
Component names should start with a capital letter and then camel casing (BirdCards)